modules/admin-ui/src/main/resources/public/org/opencastproject/adminui/languages/lang-en_US.json

```diff
@@ -1922,13 +1923,50 @@
        }
      }
    },
-   "DASHBOARD":{
+   "DASHBOARD": {
        "RUNNING":   "Running",
        "FINISHED":  "Finished",
        "PAUSED" :   "Paused",
        "FAILED" :   "Failed",
        "FINISHED_WITH_COMMENTS": "Todo",
        "SCHEDULED" : "Scheduled",
        "RECORDING" : "Recording"
+   },
+   "HOTKEYS": {
+       "CHEAT_SHEET": {
+           "TITLE": "Keyboard shortcuts"
+       },
+       "KEYS": {
+           "CHEAT_SHEET": "Show/hide this help menu",
+           "GENERAL": {
+               "SELECT_NEXT_DASHBOARD_FILTER": "Select next dashboard filter",
+               "SELECT_PREVIOUS_DASHBOARD_FILTER":  "Select previous dashboard filter",
+               "REMOVE_FILTERS": "Remove filters",
```

Comment:
The key MAIN_MENU seems missing here (at least the cheat cheat misses it)

Reply:
Yip, missed that.